metrics/ddl.go

```diff
@@ -105,6 +105,14 @@ var (
 			Name:      "worker_operation_total",
 			Help:      "Counter of creating ddl/worker and isowner.",
 		}, []string{LblType})
+
+	AddIndexTotalCounter = prometheus.NewCounterVec(
```

Comment:
If we use a counter, it never goes down, right?

Reply:
No, It will. `sum(rate(tidb_ddl_add_index_total[1m])) by (type)`

![image](https://user-images.githubusercontent.com/26020263/65575396-482b3d00-dfa2-11e9-9ba7-e58d88615853.png)

Reply:
I see